### ISSUE TITLE
[java] UnnecessaryFullyQualifiedName doesn't detect java.lang FQ names as violations

### ISSUE THREAD

**Issue by jsotuyod** [pmd/pmd#1186]
**Affects PMD Version:** 6.4.0

**Rule:**
[`category/java/codestyle.xml/UnnecessaryFullyQualifiedName`](https://pmd.github.io/pmd-6.4.0/pmd_rules_java_codestyle.html#unnecessaryfullyqualifiedname)

**Description:**
When the import is implicit rather than explicit, unnecessary FQCN uses are not flagged.

Possibly the same error occurs for same-package classes.

**Code Sample demonstrating the issue:**

```
public class JavaLang {
    public void convert(java.lang.String s) {
    }
}
```

**Running PMD through:** Any

**Comment by jsotuyod** [comment]
The `java.lang` issue has been fixed, the same-package hasn't been yet. I'm opening a new issue to track it.
